controllers/constants/constants.go

```diff
@@ -24,13 +24,17 @@ const (
 	IstioIngressServiceHTTPPortName  = "http2"
 	IstioIngressServiceHTTPSPortName = "https"
 	IstioSidecarInjectAnnotationName = "sidecar.istio.io/inject"
+	KserveNetworkVisibility          = "networking.kserve.io/visibility"
 
 	LabelAuthGroup     = "security.opendatahub.io/authorization-group"
 	LabelEnableAuthODH = "security.opendatahub.io/enable-auth"
```

Comment:
Here's a reworded version:
IMO
Instead of using enable-auth, it should be set to true. This will appear as "networking.kserve.io/visibility": "true". 
 how to change the status if they want to use disable-auth with current node  ?

Reply:
To disable route - `networking.kserve.io/visibility : cluster-local` 
To disable auth - remove label `security.opendatahub.io/enable-auth`